lib/src/configuration.cc

```diff
@@ -84,17 +84,15 @@ namespace lth_util = leatherman::util;
         }
     }();
 
-    // TODO(task) switch back to tasks-cache when downloading is implemented.
-    static const std::string DEFAULT_TASK_CACHE_DIR { (DATA_DIR / "tasks").string() };
+    static const std::string DEFAULT_TASK_CACHE_DIR { (DATA_DIR / "tasks-cache").string() };
```

Comment:
I'd suggest making task_cache_dir configurable so we can run as non-root and override for local testing.